tests/php/src/Helpers/ValidationRequestMocking.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace AmpProject\AmpWP\Tests\Helpers;
+
+use AMP_Validated_URL_Post_Type;
+use WP_Query;
+
+/**
+ * Class ValidationRequestMocking
+ *
+ * Helpers for validation tests.
+ */
+final class ValidationRequestMocking {
```

Comment:
Would this be maybe better style as a trait so static methods aren't required?

Reply:
Done in 2e484276cea28486f5366917ee7a989b84a26958